mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -391,6 +403,10 @@ pub async fn create_wallet_db(
                     )?;
                 }
             }
+
+            if db.get_extended_full_viewing_keys()?.is_empty() {
+                init_accounts_table(&db, &[evk])?;
+            }
```

Comment:
Why was this separated? I guess separation of the conditions in the above comment is better if it causes no problems.

Reply:
same answer as above